packages/simcore-sdk/src/simcore_sdk/node_ports_common/client_session_manager.py

```diff
@@ -17,11 +17,14 @@ def __init__(self, session=None):
         # there is no timeout for file download operations
 
         self.active_session = session or ClientSession(
+            connector=TCPConnector(
+                force_close=True
+            ),  # NOTE: this disable keep-alive connections, this might be a potential fix for https://github.com/ITISFoundation/osparc-simcore/issues/3531
```

Comment:
are you sure this is the right link to issues?

Reply:
yes

Reply:
https://github.com/ITISFoundation/osparc-simcore/issues/3531